wazo_calld/plugins/transfers/ari_helpers.py

```diff
@@ -137,3 +140,38 @@ def convert_transfer_to_stasis(
         exten='transfer',
         extra_channel=initiator_call,
     )
+
+
+def get_initial_transfer_variables(
+    ari: ARIClientProxy, initiator_call: str
+) -> tuple[str, str, dict, int | None]:
+    """
+    Dual of convert_transfer_to_stasis above: extract variables injected into initiator_channel
```

Comment:
What do you mean "dual"?

Reply:
https://en.wikipedia.org/wiki/Duality_(mathematics)
"Counterpart", "Complement" are perhaps better suited here.